docs/ReleaseNotes.md

```diff
@@ -1,6 +1,42 @@
 # Acuminator Release Notes
 This document provides information about fixes, enhancements, and key features that are available in Acuminator.
 
+## Acuminator 2.0
+Acuminator 2.0 includes the bug fixes and enhancements described in this section, as well as the features that have been implemented in previous versions.
+
+### Key Changes
+Acuminator 2.0 includes the following key changes:
+
+ - Acuminator no longer supports Visual Studio 2015. 
+ - Acuminator now supports C# 7.0 and 8.0.
```

Comment:
@k-s-popova  no, no C# 8. It's not even released yet. Just C# 7

Reply:
fixed